mlrun/feature_store/feature_set.py

```diff
@@ -617,8 +617,19 @@ def to_dataframe(
         start_time=None,
         end_time=None,
         time_column=None,
+        **kwargs,
     ):
-        """return featureset (offline) data as dataframe"""
+        """return featureset (offline) data as dataframe
+
+        :param columns:      list of columns to select (if not all)
+        :param df_module:    DataFrame class (pd for Pandas, dd for Dask, ..)
```

Comment:
I think this message is a bit confusing. One might think it's a string. Also it's similar but slightly different from the doc string in two other places.

Reply:
updated